<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewBill</class>
 <widget class="QDialog" name="NewBill">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Bill</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>92</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>92</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>90</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>326</x>
     <y>90</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>311</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>570</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Finish</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>132</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>134</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Client:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>70</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>            Add Meal</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>70</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>            Add Drink</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
